research/picai/README.md

```diff
@@ -94,35 +94,82 @@ The raw annotations differ in spatial resolution across annotators. Specifically
+The PICAI competition has prepared pre-determined 5-fold cross-validation splits of all 1500 cases. The splits do not contain patient overlap between train and validation splits. A json file, corresponding to the **splits_path** argument above, was released by the PICAI challenge which specifies the samples involved in each of the splits. This file has been made available on the cluster at
+
+```
+/ssd003/projects/aieng/public/PICAI/input/splits.json
+```
 
+If **splits_path** is passed to the `prepare_data.py` script, it will generate two files for each split. One is a json file containing pairs of scan paths and their annotation path for the train portion of the fold. On the cluster these files are available at:
 ```
-/ssd003/projects/aieng/public/PICAI/workdir/results/UNet/overviews/Task2203_picai_baseline/PI-CAI_train-fold-<fold_id>.json
+/ssd003/projects/aieng/public/PICAI/nnUNet/nnUNet_raw/Dataset003_PICAI/overviews/train-fold-<fold_id>.json
 ```
 
 Similarly, the validation dataset file for a specific fold is available at:
 ```
-/ssd003/projects/aieng/public/PICAI/workdir/results/UNet/overviews/Task2203_picai_baseline/PI-CAI_val-fold-<fold_id>.json
+/ssd003/projects/aieng/public/PICAI/nnUNet/nnUNet_raw/Dataset003_PICAI/overviews/val-fold-<fold_id>.json
 ```
 
 To learn about the details of the preprocessing, please refer to the [General Setup Documentation](https://github.com/DIAGNijmegen/picai_baseline?tab=readme-ov-file#general-setup) and subsequently the [U-Net Baseline Data Preparation](https://github.com/DIAGNijmegen/picai_baseline/blob/main/unet_baseline.md#u-net---data-preparation).
+
+### Prostate Whole Gland Segmentation
+Another related segmentation task using the aforementioned dataset is Prostate Whole Gland (WG) Segmentation. As the name implies, WG Segmentation involves segmenting the entire prostate, consisting of the peripheral zone (PZ) and the transition zone (TZ) ([more on this](https://radiopaedia.org/articles/prostate)). WG Segmentation is an important task in itself but is often used as a means to localize the prostate to register between sequences or narrow the region of interest for downstream segmentation tasks. The PICAI grand challenge has provided [AI-derived segmentation maps of the WG](https://github.com/DIAGNijmegen/picai_labels) for all of the samples in the dataset. To this end, a dataset with the identical preprocessing as above (except for the fact it only includes T2 sequence) is made available on the cluster.
+
+Each preprocessed image sample is stored on the cluster at the following path:
+```
+/ssd003/projects/aieng/public/PICAI/nnUNet/nnUNet_raw/Dataset001_PICAI_WG/imagesTr/
+```
+
+and the corresponding label
+```
+/ssd003/projects/aieng/public/PICAI/nnUNet/nnUNet_raw/Dataset001_PICAI_WG/labelsTr/
```

Comment:
Do these have the same file name format as that documented in (for example) lines 132 and 142? Worth mentioning if so.